src/ride_gui.cpp

```diff
@@ -26,7 +26,7 @@
 class RideSelectGui : public GuiWindow {
 public:
 	RideSelectGui();
-	~RideSelectGui();
+	~RideSelectGui() = default;
```

Comment:
Just for my curiosity, is this different from simply deleting the line entirely?
(If not, why keep the line?)

Reply:
I've heard that some compilers can optimize better if ctors and dtors are explicitly defaulted than if they have an empty body or are missing. (I can't think of examples right now though, so if you prefer I could also remove the line.)

Reply:
I checked this again and apparently only empty dtor bodies are a problem for optimizing. Removed the line.